BragiCode/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -52,32 +51,33 @@ public void RunIntakeState() {
   }
 
   public void WaitingForBall() {
-    IntakeM1.set(0.1);
-    IntakeM2.set(0.1);
+    IntakeMotorRight.set(0.1);
+    IntakeMotorLeft.set(0.1);
 
   }
 
   public void PopOutIntake() {
-    IntakeM3.setPosition(0);
-    if (IntakeM1.getFault_ForwardHardLimit().getValue()) {
+    IntakePopMotor.setPosition(0);
+    if (IntakeMotorRight.getFault_ForwardHardLimit().getValue()) {
       mIntakeState = IntakeState.S_WaitingForShooter;
     }
   }
 
   public void WaitingForShooter() {
-    IntakeM1.set(0);
-    IntakeM2.set(0);
+    IntakeMotorRight.set(0);
+    IntakeMotorLeft.set(0);
   }
 
-  public double IntakePos(TalonFX motorFx)
-  {
+  private static double IntakePos(TalonFX motorFx) {
     return motorFx.getPosition().getValueAsDouble();
   }
-  public void Position(){
-    SmartDashboard.putNumber("IntakeMotor1Position",IntakePos(IntakeM1));
-    SmartDashboard.putNumber("IntakeMotor2Position",IntakePos(IntakeM2));
-    SmartDashboard.putNumber("IntakeMotor3Position",IntakePos(IntakeM3));
+
+  public void Position() {
+    SmartDashboard.putNumber("IntakeMotor1Position", IntakePos(IntakeMotorRight));
+    SmartDashboard.putNumber("IntakeMotor2Position", IntakePos(IntakeMotorLeft));
+    SmartDashboard.putNumber("IntakeMotor3Position", IntakePos(IntakePopMotor));
```

Comment:
Give the dashboard descriptive names.

The dashboard text is a label. We can use Title Case. For example `Intake Motor Pop`. Looks spaces and everything!